tasks/main.yml

```diff
@@ -13,8 +13,12 @@
     - "setup-{{ ansible_pkg_mgr }}.yml"
     - "setup-default.yml"
 
-- name: Ensure Cockpit Web Console is started and enabled on boot.
+- name: Ensure Cockpit Web Console is started/stopped based on cockpit_started variable
   service:
     name: "{{ __cockpit_daemon }}"
-    state: started
-    enabled: yes
```

Comment:
can you do one service task that both starts/stops and enables/disables cockpit?
```
 - name: Ensure Cockpit Web Console is started/stopped and enabled/disabled
   service:
     - name: "{{ __cockpit_daemon }}"
     - enabled: "{{ cockpit_enabled | bool }}"
     - state: "{{ 'started' if cockpit_started | bool else 'stopped' }}"
```
?

Reply:
Yes, what would you recommend the variable be named to control starting/stopping and enabling/disabling?   cockpit_enabled?

Reply:
I think having two separate variables is fine, and the variables you have are fine.

Reply:
Updated to just use one task.